src/sentry/api/endpoints/organization_user_teams.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.base import DocSection
+from sentry.api.serializers import serialize
+from sentry.models import Team, TeamStatus
+
+
+class OrganizationUserTeamsEndpoint(OrganizationEndpoint):
+    doc_section = DocSection.TEAMS
+
+    def get(self, request, organization):
+        """
+        List your Teams In the Current Organization
+        ```````````````````````````````````````````
+
+        Return a list of the teams available to the authenticated session and
+        with the supplied organization. If the user is a super user, then return
+        all teams within the organization.
+
+        @param string super_user: specify "1" to request as a super user
+        """
+        super_user = request.GET.get("super_user")
+        if super_user is not None and super_user == "1":
```

Comment:
Oh, I misunderstood what you were asking. You should just use
```
from sentry.auth.superuser import is_active_superuser
is_active_superuser(request)
```
And then return all the teams if the user is a super user. No need for the super user param.

Reply:
ahh gotcha thanks will change